client/sites/components/sites-dataviews/dataviews-fields/site-field.tsx

```diff
@@ -77,45 +122,31 @@ const SiteField = ( { site, openSitePreviewPane }: Props ) => {
 
 	const isAdmin = useSelector( ( state ) => canCurrentUser( state, site.ID, 'manage_options' ) );
 
-	const onSiteClick = ( event: React.MouseEvent ) => {
-		event.preventDefault();
-
-		// Ignore if not left or middle click
-		if ( event.button > 1 ) {
-			return;
-		}
-
-		let openInNewTab = false;
-		if ( event.ctrlKey || event.metaKey ) {
-			openInNewTab = true;
-		}
-		// Support middle click to open in new tab
-		if ( event.button === 1 ) {
-			openInNewTab = true;
-		}
+	// TODO: Consolidate with `isActionEligible` logic in actions.tsx
+	const shouldOpenSitePreviewPane =
+		isAdmin &&
+		! isP2Site &&
+		! isNotAtomicJetpack( site ) &&
+		! isDisconnectedJetpackAndNotAtomic( site );
 
-		if (
-			isAdmin &&
-			! isP2Site &&
-			! isNotAtomicJetpack( site ) &&
-			! isDisconnectedJetpackAndNotAtomic( site )
-		) {
-			openSitePreviewPane && openSitePreviewPane( site, 'site_field', openInNewTab );
+	const onSiteClick = ( shouldOpenNewTab: boolean ) => {
+		if ( shouldOpenSitePreviewPane ) {
+			sitePreviewPane.open( site, 'site_field', shouldOpenNewTab );
 		} else {
-			navigate( adminUrl, openInNewTab );
+			navigate( adminUrl, shouldOpenNewTab );
 		}
 	};
 
 	const isMigrationPending = getMigrationStatus( site ) === 'pending';
 	const siteTitle = isMigrationPending ? translate( 'Incoming Migration' ) : site.title;
 
 	return (
-		<Button
+		// TODO: Consolidate behavior with `SiteIcon` link
```

Comment:
👌

Reply:
No, `SiteIcon` is the icon at the very left of each row. These are wrapped in yet another faux link 😅

<img width="114" alt="Site Icons" src="https://github.com/user-attachments/assets/35e6b441-29b5-49cc-a91c-2faccb9d01e9" />

Reply:
I'm hoping we can, given these are all UX issues. Not sure when though, given the shifting priorities.